packages/lodestar/src/node/nodejs.ts

```diff
@@ -118,13 +119,13 @@ export class BeaconNode {
       rpc,
       logger: logger.child(this.conf.logger.sync),
     });
-    //TODO: needs to be moved to Rpc class and initialized from opts
-    this.rpc = new JSONRPC(this.conf.api, {
-      transports: [new WSServer(this.conf.api.transports[0])],
-      apis: this.conf.api.apis.map((Api) => {
-        return new Api({}, {config, chain: this.chain, db: this.db, eth1: this.eth1});
-      })
-    });
+    // //TODO: needs to be moved to Rpc class and initialized from opts
+    // this.rpc = new JSONRPC(this.conf.api, {
+    //   transports: [new WSServer(this.conf.api.transports[0])],
+    //   apis: this.conf.api.apis.map((Api) => {
+    //     return new Api({}, {config, chain: this.chain, db: this.db, eth1: this.eth1});
+    //   })
+    // });
```

Comment:
why commented?

Reply:
just saw the other comment

Reply:
it doesn't work, it will get fixed in Rest API PR, we have wrong rpc api as well